packages/front-end/components/Features/SDKEndpoints.tsx

```diff
@@ -93,71 +92,58 @@ const SDKEndpoints: FC<{
               const endpoint = getSDKEndpoint(key.key, selectedProject);
 
               const envExists = environments?.some((e) => e.id === env);
+
               return (
                 <tr key={key.key}>
-                  <td>{key.description}</td>
-                  <td>
+                  <td className="d-flex flex-column">
                     <Tooltip
                       body={
                         envExists
                           ? ""
                           : "This environment no longer exists. This SDK endpoint will continue working, but will no longer be updated."
                       }
                     >
-                      {env}{" "}
+                      <strong className="mr-1">{env}</strong>
                       {!envExists && (
                         <FaExclamationTriangle className="text-danger" />
                       )}
                     </Tooltip>
+                    <span style={{ fontSize: "87.5%", fontStyle: "italic" }}>
+                      {key.description}
+                    </span>
                   </td>
                   <td>
-                    <code>{endpoint}</code>{" "}
-                    <FaCopy
-                      className="cursor-pointer"
-                      title="Copy to Clipboard"
-                      onClick={(e) => {
-                        e.preventDefault();
-                        navigator.clipboard
-                          .writeText(endpoint)
-                          .then(() => {
-                            console.log("Copied!");
-                          })
-                          .catch((e) => {
-                            console.error(e);
-                          });
-                      }}
-                    />
+                    <ClickToCopy valueToCopy={endpoint}>
+                      <span style={{ overflowWrap: "anywhere" }}>
+                        {endpoint}
+                      </span>
+                    </ClickToCopy>
                   </td>
                   {hasEncryptedEndpoints && (
-                    <td>
-                      {key.encryptSDK ? (
-                        canManageKeys ? (
-                          <ClickToReveal
-                            valueWhenHidden="Reveal key"
-                            getValue={async () => {
-                              const res = await apiCall<{
-                                key: ApiKeyInterface;
-                              }>(`/keys/reveal`, {
+                    <td style={{ width: 295 }}>
+                      {canManageKeys && key.encryptSDK ? (
+                        <ClickToReveal
+                          valueWhenHidden="Click to reveal encryption key"
```

Comment:
This is minor, but this looks slightly weird when it's blurred since you can clearly see the word breaks.  We could have the default text be something that looks like an API key instead, like `secret_abcdefghijklmnop123456789`

Reply:
I went back and forth on this - I was thinking it might be good from an accessibility standpoint to have the blurred text offer some context, but I agree it does look a bit odd. Updating now.